pkg/systemd/overview-cards/uncleanShutdownStatus.jsx

```diff
@@ -0,0 +1,98 @@
+ */
+
+import React, { useState } from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
+import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex/index.js";
+import { Icon } from "@patternfly/react-core/dist/esm/components/Icon/index.js";
+import { ExclamationTriangleIcon } from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+
+import './uncleanShutdownStatus.scss';
+import { useInit } from 'hooks';
+
+const _ = cockpit.gettext;
+
+export const UncleanShutdownStatus = () => {
+    const [uncleanShutdownId, setUncleanShutdownId] = useState(null);
+    const [uncleanShutdownStatusVisible, setUncleanShutdownStatusVisible] = useState(false);
+
+    useInit(() => {
+        cockpit.spawn(
+            ["last", "--system", "--limit=2", "--time-format=iso", "--tab-separated", "shutdown", "reboot"],
+            { environ: ["LC_ALL=C"], err: "ignore", }
+        ).then((data) => {
+            const previous_boot = data.split("\n")[1];
+            if (previous_boot === undefined)
+                return;
+
+            const fields = previous_boot.split("\t");
+            if (fields.length !== 5)
+                return;
+
+            if (fields[4].trim() !== "still running") {
+                return;
+            }
+
+            const timestamp = fields[3].trim();
+            setUncleanShutdownId(timestamp);
+            setUncleanShutdownStatusVisible(timestamp != cockpit.localStorage.getItem("dismissed-unclean-shutdown-id"));
+        });
+    });
+
+    if (!uncleanShutdownId || !uncleanShutdownStatusVisible) {
+        return null;
+    }
+
+    function hideAlert() {
+        setUncleanShutdownStatusVisible(false);
+        cockpit.localStorage.setItem('dismissed-unclean-shutdown-id', uncleanShutdownId);
+    }
+
+    return (
+        <li className="unclean-shutdown-status" id="page_status_unclean_shutdown">
+            <Flex flexWrap={{ default: 'nowrap' }}>
+                <FlexItem>
+                    <Icon status="danger">
+                        <ExclamationTriangleIcon className="system-information-unclean-shutdown-status-icon" />
+                    </Icon>
+                </FlexItem>
+                <div>
+                    <div className="pf-v6-u-text-break-word system-information-unclean-shutdown-status">
+                        {_("Unclean shutdown")}
+                    </div>
+                    <Flex>
+                        <Button variant="link" isInline
+                                    className="pf-v6-u-font-size-sm"
+                                    onClick={() => cockpit.jump("/system/logs#/?boot=-1")}>
+                            {_("View logs")}
+                        </Button>
+                        <Button variant="link" isInline
+                                    className="pf-v6-u-font-size-sm"
+                                    onClick={() => hideAlert()}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22289-c45ae717-20251111-201844-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)